app/views/dataverse/datasets/show.html.erb

```diff
@@ -6,7 +8,11 @@
       <%= render partial: 'dataverse/datasets/dataset_metadata', locals: { dataset: @dataset } %>
     </div>
     <div class="col-md-9">
-      <%= render partial: 'dataverse/datasets/dataset_files', locals: { files: @files, dataverse_url: @dataverse_url, dataset: @dataset} %>
+        <% if validation_result.valid %>
+          <%= render partial: 'dataverse/datasets/dataset_files', locals: { files: @files, dataverse_url: @dataverse_url, dataset: @dataset} %>
+        <% else %>
+          <%= validation_result.message %> 
+        <% end %>
```

Comment:
Instead of displaying the message as a free text, I would envelop the message inside a card component, like when we display a message when there are no files in the dataset. Something like this: 

<img width="1364" alt="image" src="https://github.com/user-attachments/assets/41dd9180-b559-4119-8fc1-260b28af2a4a" />

Reply:
Fixed